content/hi/docs/reference/_index.md

```diff
@@ -0,0 +1,90 @@
+---
+title: संदर्भ
+approvers:
+- chenopis
```

Comment:
```suggestion
```

Reply:
Thanks for letting me know! Removed in latest commit.